eZ/Publish/Core/REST/Client/URLAliasService.php

```diff
@@ -75,9 +75,7 @@ public function setSession($id)
      * @param bool $forwarding if true a redirect is performed
      * @param bool $alwaysAvailable
      *
-     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if the path already exists for the given language
-     *
-     * @return \eZ\Publish\API\Repository\Values\Content\URLAlias
+     * @throws \Exception
```

Comment:
This change seems to be invalid...

Reply:
> This change seems to be invalid...

This is quite valid, otherwise the issue is reported
![image](https://user-images.githubusercontent.com/7099219/89882425-1c6b7280-dbc7-11ea-85ec-ccbf1bb9e152.png)
Implementation-wise those exceptions are never thrown. They are reported on API, which is enough.